test/files/neg/t9668.scala

```diff
@@ -0,0 +1,6 @@
+object Test {
+  type Foo = String
+  object Foo
+
+  Foo : Serializable
```

Comment:
Type aliases don't have companion objects.

I still see
```
class C extends Serializable ; object C ; C : Serializable
```
Is that wrong? Maybe it is wrong. Or maybe it's a feature.